cmd/tealdbg/debugger_test.go

```diff
@@ -142,19 +143,17 @@ func TestSession(t *testing.T) {
 	err = s.SetBreakpoint(2)
 	require.NoError(t, err)
 
-	sig := make(chan struct{})
 	ackCount := 0
+	done := make(chan struct{})
 	ackFunc := func() {
-		sig <- struct{}{}
 		<-s.acknowledged
 		ackCount++
-		<-sig
+		done <- struct{}{}
```

Comment:
if would be more reliable to use defer statement at the beginning of the function, as it handles panic use cases as well.

Reply:
I do not think panic needs to be handled here, it the test panics let it print the dump